sdk/servicebus/Azure.Messaging.ServiceBus/samples/README.md

```diff
@@ -0,0 +1,17 @@
+﻿---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-service-bus
+name: Azure.Messaging.ServiceBus samples for .NET
+description: Samples for the Azure.Messaging.ServiceBus client library
+---
```

Comment:
I think this is not formatted correctly. It looks like this:
![image](https://user-images.githubusercontent.com/45216704/78182028-3d9ea400-741a-11ea-8901-ac529b1a9a26.png)
Did you mean something like this?
![image](https://user-images.githubusercontent.com/45216704/78181848-e1d41b00-7419-11ea-8c10-c0eeee5ee046.png)

Reply:
Yes.. how did you get it to format like that?

Reply:
I think your code works fine. My formatter was not parsing correctly.

Reply:
It works fine now.

Reply:
After @pakrym's commit, it works fine now.